src/algebra/curves/alt_bn128/alt_bn128_g1.hpp

```diff
@@ -8,8 +8,8 @@
 #ifndef ALT_BN128_G1_HPP_
 #define ALT_BN128_G1_HPP_
 #include <vector>
-#include "algebra/curves/alt_bn128/alt_bn128_init.hpp"
-#include "algebra/curves/curve_utils.hpp"
+#include "alt_bn128_init.hpp"
+#include "../curve_utils.hpp"
```

Comment:
'#include' names must not refer to `../` because this breaks directory scoping. For example, because all include paths are searched, including `/usr/include`, the above tells gcc to look for `/usr/curve_utils.hpp`